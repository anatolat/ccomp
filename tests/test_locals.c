
void test_params(int a, int b) {
	printf("%d %d\n", a, b);
}


int main() {
	int local = 42;
	printf("%d\n", local);

	int a;
	a = 100;
	printf("%d\n", a);

	test_params(5, 7);
}