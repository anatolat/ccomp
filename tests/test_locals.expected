42
100
5 7
